### PR TITLE
Pull Request for Desktop version portfolio

### PR DESCRIPTION
**Pull request by Tarto-4**
# Changes
Linter configuration.
Added animations
Add media quarries in CSS file

*To add transition and media quarries*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+        </a>
+        <nav class="main-nav">
+            <ul class="main-nav-list">
+                <li><a class="main-nav-link" href="#">Portfolio</a></li>
+                <li><a class="main-nav-link" href="#">About</a></li>
+                <li><a class="main-nav-link" href="#">Contact</a></li>
+                <li>
+                    <button class="mail-icon">
+<img src="img/icon-mail.svg" alt="Mail Icon" />
+</button>
```
Comment (nahnahdev) [inline index.html:25]: - [x] Great job so far @Tarto-4 You've done a good job with this project. There are just a few changes that need to be made and you'll be n your way to the next project :wink: 
- [x] It looks like throughout your project, most of your images have broken links. Which is making your project not look exactly like that of the Figma design :+1: 
- [x] In your navbar, this image isn't showing so please fix it :+1: 
- Your Design
image

- The Figma Design 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="headline-img-box"></div>
+        </div>
+    </section>
+
```
Comment (nahnahdev) [inline index.html:83]: - [x] In the headline section, you are currently missing the background image as seen in the Figma design. Kindly add it and ensure that it is well positioned as in the Figma design :100: 
- [x] Secondly, from the screenshot below, you can see that all the `social icons` are currently broken so kindly fix those as well :wink: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Javascript</li>
+                        <li>css</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
```
Comment (nahnahdev) [inline index.html:120]: - [x] In the works section, two of the image links are broken. I have indicated them in the screenshot below so kindly fix those as well :+1: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section padding-left-right">
+        <div class="about-box">
+            <div class="about-text-box">
+                <h2 class="about-heading">About me</h2>
+                <p class="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (nahnahdev) [inline index.html:203]: - [x] In the `About section` you're missing the background image seen in the Figma design :+1: I will suggest that you fix that as well :wink: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+                <ul class="categories-list">
+                    <li>Javascript</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+
+            <div class="categories">
+                <div class="categories-heading-box">
+                    <img src="img/framework-icon.svg" alt="rectangle icon" />
```
Comment (nahnahdev) [inline index.html:228]: - [x] The images for each of the categories are also currently broken so kindly fix that as well :wink: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,309 @@
...
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <img class="icon" src="img/icon-medium.svg" alt="mercedes social icon" />
+                    </a>
+                </li>
+            </ul>
+        </div>
```
Comment (nahnahdev) [inline index.html:305]: - [x] Lastly, the `social icons ` in the footer section are also currently not showing. So kindly fix that as well :100: 
- Your Design
image

- The Figma Design
image

File: index.html (hunk 4 above)
Comment (nahnahdev) [inline index.html:203]: - [x] In the mobile view, I noticed that your `About section` looks different than how it was before and how it is in the Figma design. SO kidly fix that as well :wink:

| Your Design | The Figma Design |
|--------|--------|
| image | image |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,309 @@
...
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-5.svg" alt="mercedes social icon" />
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="headline-img-box"></div>
```
Comment (nahnahdev) [inline index.html:80]: - [x] In the headline section of the mobile view, you're also missing the background image as seen in the Figma design. So kindly fix this as well :+1: 
- [x] Lastly, in the mobile view please ensure that there are no broken image links when on mobile view as well :wink: 

| Your Design | The Figma Design |
|--------|--------|
|image | image |

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Tarto-4 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No linter errors :heavy_check_mark: 
- Good use of GitHub Flow :heavy_check_mark: 
- Professional Pull request :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Tarto-4, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Images now render as expected

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**